docker-compose.yml

```diff
@@ -100,3 +100,14 @@ services:
       - cachito-api
       - nexus
       - rabbitmq
+
+  flower:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-flower
+    environment:
+      RABBITMQ_MNG: cachito:cachito@cachito_rabbitmq_1
+    # Override the default command so that Flower get access to the broker and tasks
```

Comment:
Nitpick: update the comment to include the waiting part (or just remove the comment entirely)